nemo/collections/nlp/models/language_modeling/megatron_base_model.py

```diff
@@ -826,6 +861,7 @@ def is_data_parallel_rank_zero(self):
 
     def _get_total_params_across_model_parallel_groups_gpt_bert(self, model):
         """Returns the total number of parameters across all model parallel groups."""
+        is_mcore_model = self.__dict__.get('mcore_gpt', False) or self.__dict__.get('mcore_bert', False)
```

Comment:
outside this PR, but I think we should make this a generic arg maybe "mcore_model" rather than having to check mcore_[gpt,bert] etc explicitly

Reply:
Yeah thats actually a good point. Ll discuss with eric